docs/adr/0012-moc-infra-cluster-deployment-methods.md

```diff
@@ -0,0 +1,47 @@
+# Cluster deployment methods for moc-infra cluster
+
+## Context and Problem Statement
+
+This document suggests options for the preferred deployment method for the **moc-infra cluster** at the MOC(Mass Open Cloud). This hub cluster will be the central openshift cluster to manage and deploy other clusters on-demand.
+
+The moc-infra cluster is a bare-metal cluster, deploying an OpenShift Container Platform on bare metal can be achieved by several methodologies.
+
+## Considered Options
+
+### UPI - User provioned indrastructure
+
+Deploy an OpenShift cluster on infrastructure that you prepare and maintain.
+
+We decided not to take this option under consideration since this method includes heavier infrastructure configuration.
+
+### IPI - Installer provisioned infrastructure
+
+Deploy an OpenShift cluster on infrastructure that the installation program provisions and the cluster maintains.
+
+* Good, because the installation is closer to a disconnected environment, and it's a good practice.
+* Good, because this method uses an external tool that is available for all.
+* Good, because this is a lightweight, comfortable and available tool to the general public.
+
+### Assisted Installer
+
+Discovery ISO that makes it easy to connect discovered hardware.
+
+* Good, because Assisted Installer is a growing Red Hat tool to install a new cluster, getting familiar with it will have future benefits.
+* Good, because this tool will be part of ACM deployment of new clusters.
+* Good, because it's a good opportunity to provided feedback to the developers.
+* Good, because can be easily automated, use API calls.
+* Good, because this method requires booting bare-metal nodes from an ISO image.
```

Comment:
Saying "requires" here makes this sound less like a "good" point and more like a  limitation. Maybe "allows" rather than "requires"?